### DISCUSSION TITLE
开源点评（2016-05-21） - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#97] [Announcements]
# 开源点评（2016-05-21） - 「开源之道」

https://opensourceway.community/posts/opensource/daily_reading/2016/2016_5/2016_5_21/

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

很明显这个心愿没有坚持住,
否则, 6年后的今天, 开源日报 早应该上市了...
